gymnasium/spaces/utils.py

```diff
@@ -184,15 +184,17 @@ def _flatten_multidiscrete(
 def _flatten_tuple(space: Tuple, x: tuple[Any, ...]) -> tuple[Any, ...] | NDArray[Any]:
     if space.is_np_flattenable:
         return np.concatenate(
-            [flatten(s, x_part) for x_part, s in zip(x, space.spaces)]
+            np.array([flatten(s, x_part) for x_part, s in zip(x, space.spaces)])
```

Comment:
Why do we need to add `np.array`?

Reply:
https://github.com/Farama-Foundation/Gymnasium/pull/327#discussion_r1104369738